site/en/guides/border-router/build.md

```diff
@@ -172,6 +190,28 @@ other service has failed to start. Check to see which:
 $ sudo systemctl --failed
 ```
 
+You can also check each service individually:
+
+```
+$ sudo service avahi-daemon status
```

Comment:
```suggestion
$ sudo service mdns status
```
Current bootstrap script (`./script/bootstrap`) is using mDNSResponder as the default mdns publisher.

Reply:
Ah - OK, I made a few updates; done.